application/common/components/passwordStore/Google.php

```diff
@@ -247,4 +267,49 @@ public function assess($employeeId, $password)
     {
         return true;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDisplayName(): string
+    {
+        return $this->displayName;
+    }
+
+    /**
+     * @param string $employeeId
+     * @return Google_Service_Directory_User The user record from Google.
+     * @throws UserNotFoundException
+     */
+    protected function getUserByEmployeeId($employeeId)
+    {
+        $usersResource = $this->getUsersResource();
+        $response = $usersResource->listUsers([
+                'domain' => $this->searchDomain,
+                'maxResults' => 2,
+                'query' => 'externalId=' . $employeeId,
+        ]);
+
+        if ($response === null || count($response['users']) === 0) {
+            \Yii::warning([
+                'action' => 'getEmailFromGoogle',
```

Comment:
Does it matter that 'getEmailFromGoogle' doesn't match the function name ('getUserByEmployeeId')  - here and on line 304.

Reply:
Good find. That's leftover from some refactoring.